wagtail/search/backends/database/mysql/mysql.py

```diff
@@ -491,9 +491,7 @@ def search(self, config, start, stop, score_field=None):
         )
         if not negated:
             index_entries = index_entries.filter(match_expression)
-            if (
-                self.order_by_relevance
-            ):  # Only applies to the case where the outermost query is not a Not(), because if it is, the relevance score is always 0 (anything that matches is excluded from the results).
+            if self.order_by_relevance:  # Only applies to the case where the outermost query is not a Not(), because if it is, the relevance score is always 0 (anything that matches is excluded from the results).
```

Comment:
```suggestion
            if self.order_by_relevance:
                # Only applies to the case where the outermost query is not a Not(),
                # because if it is, the relevance score is always 0
                # (anything that matches is excluded from the results).
```